personal-portfolio/index.html

```diff
@@ -4,356 +4,442 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" media="screen and (min-width: 1000px)" href="styles/reset.css">
-    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="stylesheet" href="css/style.css" />
     <link
-      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
       rel="stylesheet"
+      type="text/css"
     />
-    <title>Milan Protic Portfolio</title>
+    <title>Personal Portfolio</title>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-QQNP4YXK46"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "G-QQNP4YXK46");
+    </script>
   </head>
+
   <body>
-    <header class="toolbar">
-      <div class="logo_placeholder">
-        <a href="index.html" class="logo">Welcome</a>
-      </div>
-      <nav class="menu">
-        <button type="button" class="btn">
-          <i class="material-icons">menu</i>
-        </button>
+    <header>
+      <nav>
+        <h3><a href="#">Welcome</a></h3>
+        <ul>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>

```

Comment:
- [ ] Your mobile menu does not disappear when any of the menu options: `Portfolio`, `About` or `Contact` is clicked or tapped. To enhance the UX (User Experience) we actually the mobile menu to vanish and jump to the selected content when a user taps or clicks on the mobile menu option: 

**Your mobile Screen (When mobile _Portfolio_ option is tapped):**
![image](https://user-images.githubusercontent.com/12745474/128346320-fdd41ea8-9381-4187-8bfb-ea04b5e41255.png)

**Your mobile Screen (When mobile _About_ option is tapped):**
![image](https://user-images.githubusercontent.com/12745474/128345914-1d717738-081d-46cb-b810-19d5b1645426.png)

**Your mobile Screen (When mobile _Contact_ option is tapped):**
![image](https://user-images.githubusercontent.com/12745474/128345998-e0fa44bf-03e1-470d-98e3-25b346ce71b3.png)

Kindly ensure to improve upon the issues listed above to enhance a better UX.